src/test/java/org/jabref/logic/formatter/FormatterTest.java

```diff
@@ -29,6 +31,18 @@ public static void setUp() {
                         Collections.emptyList(), Collections.emptyList()));
     }
 
+    /**
+     * When a new formatter is added by copy and pasting another formatter, it may happen that the <code>getKey()</code> method is not adapted. This results in duplicate keys, which this test tests for.
+     */
+    @Test
+    public void allFormatterKeysAreUnique() {
+        // idea for uniqueness checking by https://stackoverflow.com/a/44032568/873282
+       assertFalse(getFormatters().collect(Collectors.groupingBy(
+               formatter -> formatter.getKey(),
+               Collectors.counting()
+       )).entrySet().stream().anyMatch(e -> e.getValue() > 1));
```

Comment:
In case somebody adds a new formatter with the same key, this test fails but the error message does not say why. I would change `anyMatch` to filter and compare (assertEquals) the resulting map to an empty map.